client/www/components/dash/explorer/Explorer.tsx

```diff
@@ -920,7 +920,7 @@ export function Explorer({
           },
         )}
       >
-        <div className="flex items-center gap-1 text-sm font-semibold">
+        <div className="flex items-center gap-1 text-sm font-semibold px-4">
           <ChevronLeftIcon
```

Comment:
This was intentional: we wanted this to be a title, and have each namespace indented under